web_timeline/static/lib/vis-timeline/vis-timeline-graph2d.min.js

```diff
@@ -0,0 +1,53 @@
length}var e,i,n=[],r=[],o=[];for(e=0;e<12;e++)i=c([2e3,e]),n.push(this.monthsShort(i,"")),r.push(this.months(i,"")),o.push(this.months(i,"")),o.push(this.monthsShort(i,""));for(n.sort(t),r.sort(t),o.sort(t),e=0;e<12;e++)n[e]=dt(n[e]),r[e]=dt(r[e]);for(e=0;e<24;e++)o[e]=dt(o[e]);this._monthsRegex=new RegExp("^("+o.join("|")+")","i"),this._monthsShortRegex=this._monthsRegex,this._monthsStrictRegex=new RegExp("^("+r.join("|")+")","i"),this._monthsShortStrictRegex=new RegExp("^("+n.join("|")+")","i")}function Gt(t,e,i,n,r,o,s){var a;return t<100&&t>=0?(a=new Date(t+400,e,i,n,r,o,s),isFinite(a.getFullYear())&&a.setFullYear(t)):a=new Date(t,e,i,n,r,o,s),a}function Ut(t){var e;if(t<100&&t>=0){var i=Array.prototype.slice.call(arguments);i[0]=t+400,e=new Date(Date.UTC.apply(null,i)),isFinite(e.getUTCFullYear())&&e.setUTCFullYear(t)}else e=new Date(Date.UTC.apply(null,arguments));return e}function zt(t,e,i){var n=7+e-i;return-(7+Ut(t,0,n).getUTCDay()-e)%7+n-1}function Vt(t,e,i,n,r){var o,s,a=1+7*(e-1)+(7+i-n)%7+zt(t,n,r);return a<=0?s=xt(o=t-1)+a:a>xt(t)?(o=t+1,s=a-xt(t)):(o=t,s=a),{year:o,dayOfYear:s}}function Bt(t,e,i){var n,r,o=zt(t.year(),e,i),s=Math.floor((t.dayOfYear()-o-1)/7)+1;return s<1?n=s+Xt(r=t.year()-1,e,i):s>Xt(t.year(),e,i)?(n=s-Xt(t.year(),e,i),r=t.year()+1):(r=t.year(),n=s),{week:n,year:r}}function Xt(t,e,i){var n=zt(t,e,i),r=zt(t+1,e,i);return(xt(t)-n+r)/7}function qt(t,e){return t.slice(e,7).concat(t.slice(0,e))}U("w",["ww",2],"wo","week"),U("W",["WW",2],"Wo","isoWeek"),I("week","w"),I("isoWeek","W"),R("week",5),R("isoWeek",5),lt("w",J),lt("ww",J,X),lt("W",J),lt("WW",J,X),pt(["w","ww","W","WW"],(function(t,e,i,n){e[n.substr(0,1)]=k(t)})),U("d",0,"do","day"),U("dd",0,0,(function(t){return this.localeData().weekdaysMin(this,t)})),U("ddd",0,0,(function(t){return this.localeData().weekdaysShort(this,t)})),U("dddd",0,0,(function(t){return this.localeData().weekdays(this,t)})),U("e",0,0,"weekday"),U("E",0,0,"isoWeekday"),I("day","d"),I("weekday","e"),I("isoWeekday","E"),R("day",11),R("weekday",11),R("isoWeekday",11),lt("d",J),lt("e",J),lt("E",J),lt("dd",(function(t,e){return e.weekdaysMinRegex(t)})),lt("ddd",(function(t,e){return e.weekdaysShortRegex(t)})),lt("dddd",(function(t,e){return e.weekdaysRegex(t)})),pt(["dd","ddd","dddd"],(function(t,e,i,n){var r=i._locale.weekdaysParse(t,n,i._strict);null!=r?e.d=r:f(i).invalidWeekday=t})),pt(["d","e","E"],(function(t,e,i,n){e[n]=k(t)}));var Zt="Sunday_Monday_Tuesday_Wednesday_Thursday_Friday_Saturday".split("_"),$t="Sun_Mon_Tue_Wed_Thu_Fri_Sat".split("_"),Jt="Su_Mo_Tu_We_Th_Fr_Sa".split("_");function Kt(t,e,i){var n,r,o,s=t.toLocaleLowerCase();if(!this._weekdaysParse)for(this._weekdaysParse=[],this._shortWeekdaysParse=[],this._minWeekdaysParse=[],n=0;n<7;++n)o=c([2e3,1]).day(n),this._minWeekdaysParse[n]=this.weekdaysMin(o,"").toLocaleLowerCase(),this._shortWeekdaysParse[n]=this.weekdaysShort(o,"").toLocaleLowerCase(),this._weekdaysParse[n]=this.weekdays(o,"").toLocaleLowerCase();return i?"dddd"===e
```

Comment:
Also push the unminified source if possible.

Reply:
Not possible (in a easy way)

https://github.com/visjs/vis-timeline
https://cdnjs.com/libraries/vis-timeline

Reply:
Oh ok.

Reply:
Can't you use https://cdnjs.cloudflare.com/ajax/libs/vis-timeline/7.3.4/vis-timeline-graph2d.esm.js ?

Reply:
I think that "ES Modules" is not that we want... is valid use "import" in Odoo modules?